integration-tests/testconfig/automation/automation.toml

```diff
@@ -58,7 +58,13 @@ spec_type="minimum"
 chainlink_node_log_level="info"
 use_prometheus=false
 
+[Reorg.Seth]
+ephemeral_addresses_number = 20
```

Comment:
@Tofel Reorg test is not migrated to seth, right? Was this meant to be for chaos?